datasets/standvastgoed/dataset.json

```
@@ -132,7 +132,8 @@
           },
           "nummeraanduidingvolgnummer": {
```

```suggestion
          "nummeraanduidingVolgnummer": {
```
otherwise it does not match with the reference in `temporal.identifier`